Examples/ArcGISToolkitExamples.xcodeproj/project.pbxproj

```diff
@@ -178,7 +178,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0800;
-				LastUpgradeCheck = 0820;
+				LastUpgradeCheck = 0930;
```

Comment:
Since you've performed the upgrade to Xcode 9.3, could you please check in the `IDEWorkspaceChecks.plist` file that Xcode creates automatically?

Reply:
Same with the ArcGISToolkit project.